src/mapping/impl/BasisFunctions.hpp

```diff
@@ -1,22 +1,17 @@
 #pragma once
 
-#if defined(__NVCC__)
-
+#ifdef __CUDA_ARCH__
```

Comment:
I would still check for the definition of CUDACC here

Reply:
Right. I wanted to revert that.